lib/src/util/map_util.dart

```diff
@@ -68,6 +68,58 @@ Map getPropsToForward(Map props, {
   return propsToForward;
 }
 
+/// Adds unconsumed props to a passed in [Map] reference ([propsToUpdate]).
+///
+/// Based upon configuration, the function will overlook [props] that are not
+/// meant to be passed on, such as non-DOM props or specified values.
+void forwardUnconsumedProps(Map props, {
+  bool omitReactProps: true,
+  bool onlyCopyDomProps: false,
+  Iterable keysToOmit,
+  Iterable<Iterable> keySetsToOmit,
+  Map propsToUpdate,
+}) {
+  if (onlyCopyDomProps) {
+    for (String key in props.keys) {
+      if (key.startsWith('aria-') ||
+          key.startsWith('data-') ||
+          _validDomProps.contains(key)) {
+        propsToUpdate[key] = props[key];
+      }
+    }
+    return;
+  }
+
+  for (String key in props.keys) {
+    if (keysToOmit != null && keysToOmit.contains(key)) continue;
+
+    if (keySetsToOmit != null) {
+        /// If the passed in value of [keySetsToOmit] comes from
+        /// [addUnconsumedProps], there should only be a single index.
+        /// Consequently, this case exists to give the opportunity for the loop
+        /// to continue without initiating another loop (which is less
+        /// performant than `.first.contains()`).
+        if (keySetsToOmit.first.contains(key)) continue;
+
+        if (keySetsToOmit.length > 1) {
+          bool shouldContinue = false;
+          for (Iterable keySet in keySetsToOmit) {
+            if (keySet.contains(key)) {
+              shouldContinue = true;
+              continue;
+            }
+          }
+
+          if (shouldContinue) continue;
```

Comment:
Is this necessary? 

There is no logic below here which would be "skipped" by continuing to the next iteration in the `for (String key in props.keys)` loop as far as I can tell - which would make this `continue;` superfluous.

Did you mean for it to break out of the loop if `shouldContinue` is false?

```dart
if (!shouldContinue) break;
```

Reply:
My intend was to skip the statements on [lines 117-119](https://github.com/Workiva/over_react/blob/50e497ee5191d5db1289b63657d60d9e81aaaa26/lib/src/util/map_util.dart#L117-L119), where the key is added to the props reference. Since the loop ends on line 120, I believe those lines would be dodged by the continue. Did I miss something and that's not actually the case?

Reply:
No - you're right. This is just another example of the pitfalls of reviewing code before 6am ;)

Reply:
Hahah yep, that would do it!